packages/hash/api/src/graphql/typeDefs/knowledge/entity.typedef.ts

```diff
@@ -98,4 +136,20 @@ export const knowledgeEntityTypedef = gql`
     """
     linkedEntities: [KnowledgeLinkedEntityDefinition!]
   }
+
+  extend type Query {
+    """
+    Get an entity.
+    """
+    knowledgeEntity(
+      """
+      The id of the entity.
+      """
+      entityId: ID!
+      """
+      The version of the entity. Defaults to the latest version.
+      """
+      entityVersion: ID
```

Comment:
The version is a `String` now (iso8601 timestamp)

Reply:
Thanks for catching, also noticed I've inconsistently referred to this as `entityVersionId`. Addressed both in https://github.com/hashintel/hash/pull/1140/commits/c918ff37d40123e68049acbd8ec8a178763a18ec